ome/omero-figure
# 3rd party versions cleanup

Add version / date to all paths of all static 3rdparty libs.
Removed unused libs.

cc @jburel 

To test:
 - Quick check that app is working correctly and no 404s or other errors (check console).
